docs/introduction/faq-and-common-issues.md

```diff
@@ -152,9 +152,9 @@ See [Outgoing emails](https://github.com/djfarrelly/MailDev#outgoing-email) in t
 ## Can I see what is really happening in the Codeception acceptance tests when using Docker?
 Yes, you can! Check [the quick guide](./running-acceptance-tests.md#how-to-watch-what-is-going-on-in-the-selenium-browser).
 
-## Why is there a faked PHP 7.2 platform in the Composer config?
-As a general rule, packages and libraries that depend on PHP 7.2 will work as expected even on PHP 7.4 (any higher 7.x version), but not vice versa.
-Mainteiners of PHP are focusing on backward-compatibility (even if there were [some incompatible changes](https://www.php.net/manual/en/migration73.incompatible.php) introduced in PHP 7.3, in practice it doesn't cause issues).
+## Why is there a faked PHP 7.4.1 platform in the Composer config?
```

Comment:
Not sure if it is right or there should be `7.4.12` instead.

Reply:
it's correct as we fake 7.4.1 in `composer.json`.

Reply:
As above, now it make sense to me!